### ISSUE TITLE
Scripts documentation

### ISSUE THREAD

**Issue by maxpietsch** [MRtrix3/mrtrix3#692]
Scripts that do not use the lib.app framework are currently not in the documentation.`foreach` and `notfound` contain a `HELP_PAGE` which could also be copied to the docs.

I would also suggest to add a brief HELP_PAGE to the other scritpts (average_response, blend, convert_bruker). This would help in the homebrew installation to identify scripts to link to /url/local/bin. Currently, I link foreach, average_response, blend, convert_bruker and notfound manually.

**Comment by Lestropie** [comment]
Alternatively, those scripts could be adapted to the library framework? That way users would get a help page just like with everything else.

**Comment by jdtournier** [comment]
I think some of those scripts could probably be moved out of the main repo, and made available via a different repo. There is no way I want to support the `convert_bruker` script to the level that we support the rest of the code, it really is there as a convenience on a best-effort basis... I'm guessing the same applies for `blend`?

**Comment by draffelt** [comment]
Agreed `convert_bruker` could go.  I was planning on writing a blog post/tutorial on the screenshot tool. Which would include a section on how to make spinning brains videos that blend different images together while rotating. 

Happy to move blend into a personal repo of mine. However I think it's a pretty neat way of presenting FBA results, so it may be nice to keep it in. 

I'm also planing on writing a blog post on batch processing images with `foreach`. Something like I used to have on our internal wiki. I guess we could reimplement both `foreach` and notfound in python if someone has spare time. But that won't be me any time soon.

**Comment by Lestropie** [comment]
Not sure I'd be removing scripts: only takes one user to be using it and discover it's disappeared to cause a complaint. I reckon just leave them there, maybe add to the docs FAQ section something like 'why aren't all scripts listed in the documentation?', and just say that these are provided in case they're useful to someone 'on good faith', but aren't fully supported. I just think any effort here would be better spent on porting them than hacking the documentation to try to include them as-is.

**Comment by SebastianoF** [comment]
Small comment about the convert_bruker and related documentation, as I just found the script by accident after having recently implemented my version of the same purpose converter. 
I believe as well it should be better advertised/documented as it is invisible from an "outside" google search for who is looking only for that function.
